index.html

```diff
@@ -169,19 +169,20 @@
 The <dfn id="netinfo.type">`type`</dfn> attribute, when getting, returns the <a title="connection types">connection type</a> that the user agent is using.
 
 ### `downlinkMax` attribute
-The <dfn>`downlinkMax`</dfn> attribute represents the <a>maximum downlink speed</a>, in <a>megabits per second</a> (Mbps), for <a>underlying connection technology</a>.
+The <dfn>`downlinkMax`</dfn> attribute represents the <a>maximum downlink speed</a>, in <a>megabits per second</a> (Mbps), of the first network hop, which is determined by the properties of the <a>underlying connection technology</a>.
```

Comment:
"represents the maximum downlink speed" -> an upper bound of the maximum downlink speed